cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/flow/b2c/CreatePairwiseFlowB2C.kt

```diff
@@ -0,0 +1,16 @@
+package com.luxoft.blockchainlab.corda.hyperledger.indy.flow.b2c
+
+import co.paralleluniverse.fibers.Suspendable
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.InitiatingFlow
+
+// TODO: this flow should create connection somehow
+object CreatePairwiseFlowB2C {
```

Comment:
Do we need that class?

Reply:
If functionality is required and not designed yet, let's just create a task?

Reply:
We should discuss as well. For now, it's future is not clear.

Reply:
I suggest to postpone it and just have a task in backlog

Reply:
https://github.com/hyperledger-labs/cordentity/projects/1#card-18735717